src/components/ApiMenu/ApiMenu.tsx

```diff
@@ -16,17 +16,20 @@ const ApiMenu = ({
   const apiFree = useStore((state) => state.apiFree);
   const setApiFree = useStore((state) => state.setApiFree);
 
+  const [selectedApiFree, setSelectedApiFree] = useState<boolean>(apiFree);
```

Comment:
let's change `selectedApiFree` to `_apiFree` and `setSelectedApiFree` to `_setApiFree`, so that it is consistent with the `_apiKey` and `_setApiKey`.